package.json

```diff
@@ -97,7 +97,7 @@
   ],
   "devDependencies": {
     "go-ipfs": "^0.9.1",
-    "ipfs": "^0.59.0",
+    "ipfs": "^0.60.0",
```

Comment:
@achingbrain how do I test the fix from  ipfs/js-ipfs#3921?

Reply:
This test has just the right number of magic file names to trigger the hashing behaviour that caused the bug fixed in that PR: https://github.com/ipfs/js-ipfs/pull/3936/files#diff-77ef56bdffd9bf9f144b53c8ca4937504b82421c88ddf508ed0d11f4d25904f3R908-R949

Reply:
Though is that the sort of thing that you would test here?  It doesn't seem like an interop issue, just a bug in js-ipfs.